Ryujinx.Ava/Modules/Updater/Updater.cs

```diff
@@ -278,7 +278,7 @@ public static async void UpdateRyujinx(Window parent, string downloadUrl)
                 {
                     string ryuName = Path.GetFileName(Environment.ProcessPath);
                     string ryuExe = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, ryuName);
-                    string ryuArg = string.Join(" ", Environment.GetCommandLineArgs().Skip(1).ToArray());
+                    string[] ryuArg = Environment.GetCommandLineArgs().Skip(1).ToArray();
```

Comment:
Cloning it is unnecessary. The function already takes an `IEnumerable`
```suggestion
                    var ryuArg = Environment.GetCommandLineArgs().Skip(1);
```
Or just skip the variable and use it directly. Same for the other.

Reply:
Fair enough. I've set both to var instead.

I'd rather leave the declaration, though. It is clear what it does and easy to understand.